lib/ui/discover/HeroListItem.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:farmsmart_flutter/ui/common/network_image_from_future.dart';
+import 'package:flutter/material.dart';
+
+import 'StandardListItem.dart';
+import 'discover_page.dart';
+
+class HeroListItem {
+  Widget builder(ArticlesItemListViewModel viewModel, {ArticleListItemStyle itemStyle = const HeroArticleListItemStyle()}) {
```

Comment:
this should be _DefaultListItem style and have the default defined in this file.